src/opencensus/proto/error/v1/error.proto

```diff
@@ -0,0 +1,245 @@
+    //  Going forward, having stack parsing implemented in one place (agent exporter)
+    //  and for specific language (i.e using helper tools that exists in this language) 
+    //  is more sustainable approach then making all backends parse all language-specific errors.
+    // 
+    //  On the other side, backends that do not need parsed stack would still need raw one 
+    //  and making them format string from it is not sustainable again.
+    
+    //  Proposal: by default send both, make it configurable by agent/service
+  }
+
+  // Error details associated with this error
+  ErrorDetails error_info = 5;
+
+  // Represents a list of exceptions that caused this exception
+  // Java: cause
+  // Python 3: cause
+  // Ruby: cause
+  // C#: InnerException
+  message CauseErrors
+  {
+    repeated ErrorDetails cause = 1;
+    // TODO: Java: what about suppressed exceptions?
+  }
+  
+  // List of exceptions that cause the error.
+  CauseErrors causes = 6;
+  
+  // Error severity.
+  ErrorSeverity severity = 7;
+
+  //TODO: can we have common code with trace proto?
+  // A set of attributes, each with a key and a value.
+  message Attributes {
+    // The set of attributes. The value can be a string, an integer, or the
+    // Boolean values `true` and `false`. For example:
+    //
+    //     "HelpLink": "http://some.link"
+    //     "custom.attribute": ""
+    map<string, AttributeValue> attribute_map = 1;
+
+    // The number of attributes that were discarded. Attributes can be discarded
+    // because their keys are too long or because there are too many attributes.
+    // If this value is 0, then no attributes were dropped.
+    int32 dropped_attributes_count = 2;
+  }
+  
+  // A list of attributes on the error
+  Attributes attributes = 8;
+}
+
+// Indicates error severity
+enum ErrorSeverity
+{
+  SEVERITY_UNSPECIFIED = 0;
+
+  // Indicates critical issue, panic, after which application stops
+  // such as critical configuration error. OutOfMemory, etc...
+  CRITICAL = 1; 
+
+  // Indicates unhandled exception that does not lead to 
+  // application crash such as unhandled exception in the controller
+  ERROR = 2; 
+
+  // Indicates handled exception. Must be reported in application code
+  // explicitly. Agent should allow configuring error verbosity and 
+  // prevent such errors from being exported
+  WARNING = 3;
+}
+
+// TODO : copy-paste from trace, move to common module
+
+// The value of an Attribute.
+message AttributeValue {
+  // The type of the value.
+  oneof value {
+    // A string up to 256 bytes long.
+    TruncatableString string_value = 1;
+    // A 64-bit signed integer.
+    int64 int_value = 2;
+    // A Boolean value represented by `true` or `false`.
+    bool bool_value = 3;
+    // A double value.
+    double double_value = 4;
+  }
+}
+
+// The call stack which originated this span.
+message StackTrace {
+  // A single stack frame in a stack trace.
+  message StackFrame {
```

Comment:
Including ordering number may be helpful. If not - at least comment on how stack trace is ordered (lowest frame first or last?).